styleguide.config.js

```diff
@@ -130,14 +130,11 @@ module.exports = {
                     ],
                 },
                 {
-                    // NOTE(jeresig): Not ready to be used yet, need docs.
-                    private: true,
```

Comment:
If this is internal only stuff that isn't part of the public API, then it should remain marked private so that it only appears in the dev server version of docs that goes up for PRs and local dev; rather than in the published public docs on the main wonder-blocks netlify.

Reply:
That's one of the big changes we are doing with v2. Devs now should be able (under certain conditions) to create their own modal variations. That's the main reason why we want to expose these building blocks. You can take a look at the source code for this example: https://deploy-preview-413--wonder-blocks.netlify.com/#!/Building%20Blocks/1

That said, I'm thinking there could be a risk of being so flexible with modal layouts, as this could cause some fragmentation in WB Modals created inside Webapp. Nefaur expressed this concern as well. Maybe we would need to revisit this.

What do you think?

Reply:
To avoid fragmentation I think some patterns could be codified such as the two-column layout which you prototyped.  That layout is used in a number of places in webapp.  One question though is whether it makes sense to update the existing two column layouts to use the new version you prototype.  That's a question for design.